README.md

```diff
@@ -25,11 +25,14 @@ https://github.com/acheong08/ChatGPT/wiki/Setup
 ```json
 {
   "session_token": "<token>",
-  "proxy": "<proxy>"
+  "proxy": "<proxy>",
+  "driver_exec_path": "./path/to/driver",
+  "browser_exec_path": "./path/to/browser",
+  "conversation": "00000000-0000-0000-0000-000000000000"
 }
 ```
 
-Proxy is optional
+All options except `session_token` are optional
```

Comment:
Email and password authentication was just added. Session token is also optional